bedrock/products/templates/products/vpn/landing.html

```diff
@@ -17,6 +17,12 @@
 {% set _utm_campaign = 'vpn-product-page' %}
 {% set _params = '?utm_source=' ~ _utm_source ~ '&utm_medium=referral&utm_campaign=' ~ _utm_campaign %}
 
+{% block experiments %}
+  {% if switch('vpn-landing-page-cta-change') %}
```

Comment:
I think one of the experiment requirements is to limit this to en-US only, so you can add a parameter for that here:
```suggestion
  {% if switch('vpn-landing-page-cta-change', ['en-US']) %}
```